catalog/gke/Kptfile

```diff
@@ -0,0 +1,84 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: gke
+packageMetadata:
+  shortDescription: GKE cluster with dedicated subnet
+openAPI:
+  definitions:
+    io.k8s.cli.setters.projects-namespace:
+      x-k8s-cli:
+        setter:
+          name: projects-namespace
+          value: projects
+    io.k8s.cli.setters.network-project-id:
+      x-k8s-cli:
+        setter:
+          name: network-project-id
+          value: network-project-id
+    io.k8s.cli.setters.networking-namespace:
+      x-k8s-cli:
+        setter:
+          name: networking-namespace
+          value: config-control
+    io.k8s.cli.setters.platform-project-id:
+      x-k8s-cli:
+        setter:
+          name: platform-project-id
+          value: platform-project-id
+    io.k8s.cli.setters.platform-namespace:
+      x-k8s-cli:
+        setter:
+          name: platform-namespace
+          value: config-control
```

Comment:
Would it be better to keep this generic as `platform -project-namespace` and in tutorials explicitly set this to `config-control` as needed?

Reply:
perhaps. That might make people less likely to use config-control, which may or may not be desirable.

Reply:
`config-control` should be the default.